website/docs/reference/operators/array-operators.md

```diff
@@ -97,11 +101,12 @@ The output:
 
 ## $elemMatch
 
-*Syntax*: `{ <field>: { $elemMatch: { <query1>, <query2>, ... } } }`
+*Syntax*: `{ <field>: { $elemMatch: { <condition1>, <condition2>, ... <conditionN>} } }`
 
-The `$elemMatch` operator matches documents where the field value is an array that contains at least one element that matches the given condition.
+To select documents in a specified array field that matches at least one of several listed query conditions, use the `$elemMatch` operator.
```

Comment:
```suggestion
To select documents in a specified array field where one or more elements match all listed query conditions, use the `$elemMatch` operator.
```

Trying to highlight that we want to find one element or more to match all conditions.